protonvpn_cli/cli.py

```diff
@@ -142,6 +142,7 @@ def cli():
         check_init()
         configure_cli()
     elif args.get("refresh"):
+        check_init()
```

Comment:
Why this `check_init()` call? Is it related to the commit?

Reply:
Not necessarily, I just noticed that this is missing while testing.

Using refresh without an initialized profile throws an error.